app/players/player.py

```diff
@@ -0,0 +1,14 @@
+from abc import ABC, abstractmethod
+
+
+class Player(ABC):
+    def __init__(self, nickname: str) -> None:
+        self.nickname = nickname
+
+    @abstractmethod
+    def get_rating(self) -> int:
+        return 0
```

Comment:
The `get_rating` method is marked as abstract but contains an implementation. Abstract methods should not have a body. Consider removing the implementation to enforce that subclasses provide their own implementation.